src/misc/lv_anim_timeline.c

```diff
@@ -186,6 +187,20 @@ static void anim_timeline_set_act_time(lv_anim_timeline_t * at, uint32_t act_tim
             if(a->exec_cb) a->exec_cb(a->var, value);
             if(a->custom_exec_cb) a->custom_exec_cb(a, value);
         }
+ 
+        if(at->reverse) {
+            if(value > a->start_value) at->anim_dsc[i].flag_completed = false;
+            else if (value == a->start_value) {
+                if(! at->anim_dsc[i].flag_completed && a->completed_cb) a->completed_cb(a); 
+                at->anim_dsc[i].flag_completed = true;
+            }
+        }else{
```

Comment:
Run scripts/code-format.py ([astyle](http://astyle.sourceforge.net/install.html) version [v3.4.12](https://github.com/szepeviktor/astyle/releases/tag/v3.4.12) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).